src/components/sections/workGallery/index.jsx

```diff
@@ -0,0 +1,83 @@
+import React, { useState, useEffect } from 'react';
+import Button from '../../units/Button';
+import Image from '../../units/Image';
+import Draw from './canvas';
+import './styles.scss';
+
+function drawCanvas(canvas) {
```

Comment:
habiamos quedado que la animacion no se iba a incluir, que eso lo agregamos hasta al final como un nice to have, porfa eliminalo de este PR.